core/src/stages/move_relative.cpp

```diff
@@ -104,6 +104,59 @@ static bool getJointStateFromOffset(const boost::any& direction, const moveit::c
 	return false;
 }
 
+static void shapeMarker(visualization_msgs::Marker& m, uint32_t shape, rviz_marker_tools::Color color_id,
+                        const Eigen::Vector3d& pos, const Eigen::Quaterniond& quat, double width, double length) {
+	rviz_marker_tools::setColor(m.color, color_id);
```

Comment:
Why do you set color before "making" the marker? I had to check to see the color's not actually reset below.